make-rules/ips.mk

```diff
@@ -302,7 +302,7 @@ $(GENERATED).p5m:	install
 	[ ! -d $(SAMPLE_MANIFEST_DIR) ] && $(MKDIR) $(SAMPLE_MANIFEST_DIR) || true
 	$(PKGSEND) generate $(PKG_HARDLINKS:%=--target %) $(PROTO_DIR) | \
 	$(PKGMOGRIFY) $(PKG_OPTIONS) /dev/fd/0 $(GENERATE_TRANSFORMS) | \
-		sed -e '/^$$/d' -e '/^#.*$$/d' -e '/^dir .*$$/d' \
```

Comment:
Could this break sample-manifest target?

Reply:
It should not.  I just moved the dir action removal from Makefile to transforms/generate-cleanup.